docs/examples/PlotActivity.rst

```diff
@@ -0,0 +1,101 @@
+
+Calculate and Plot Activity
+===========================
+
+Given an existing database for Al-Zn, we would like to calculate the
+activity of the liquid. In order to make sure we are correct, we'll
+compare the values with experimental results.
+
+Set up the database
+~~~~~~~~~~~~~~~~~~~
+
+.. code:: ipython3
+
+    from pycalphad import Database, equilibrium, variables as v
+    import numpy as np
+    
+    dbf = Database('alzn_mey.tdb') 
+    
+    comps = ['AL', 'ZN', 'VA']
+    phases = list(dbf.phases.keys())
+
+Calculate the reference state
+-----------------------------
+
+You could also include this in the equilibrium calculation in the next
+step and select out the point you want, but it might be clear to be more
+explicit.
+
+.. code:: ipython3
+
+    ref_eq = equilibrium(dbf, comps, phases, {v.P: 101325, v.T: 1023, v.X('ZN'): 1})
+
+Calculate the equilibria
+------------------------
+
+Do the calculation over the compoisition range
+
+.. code:: ipython3
+
+    eq = equilibrium(dbf, comps, phases, {v.P: 1013325, v.T: 1023, v.X('ZN'): (0, 1, 0.005)})
```

Comment:
That's a pretty fine step size. Is performance acceptable?

Reply:
Yeah it's pretty fast. Takes about 3s on my machine (JIT was in previous step).